### PR TITLE
Fixed erroneous edge support data for some mask related CSS properties

### PR DESCRIPTION
**Pull request by lukewarlow**
#### Summary

Fixed erroneous edge support data for some mask related CSS properties

#### Test results and supporting details

Just changed Edge to reflect that version 79 onward matches Chromes support data.

### CODE REVIEW

File: css/properties/mask.json [hunk 1]
```diff
@@ -30,9 +30,17 @@
                 "notes": "The prefixed property can be used with SVG and HTML with a slightly different syntax, which allows setting the non-standard <a href='https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-attachment'><code>-webkit-mask-attachment</code></a> property."
               }
             ],
-            "edge": {
-              "version_added": "12"
-            },
+            "edge": [
+              {
+                "prefix": "-webkit-",
```
Comment (foolip) [inline css/properties/mask.json:35]: There's also a `partial_implementation` entry for Chrome without a prefix, and I can confirm with https://mdn-bcd-collector.appspot.com/tests/css/properties/mask that *something* is supported. Is the "While the property is recognized, values applied to it don't have any effect." note there accurate, and if so can you mirror it to Edge as well?

Comment (lukewarlow) [reply css/properties/mask.json:40]: Mirrored that across to Edge.

Comment (foolip) [reply css/properties/mask.json:40]: I've confirmed with https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9481 that it still doesn't work. https://bugs.chromium.org/p/chromium/issues/detail?id=432153 is the bug for this. I'll add that to the notes.

Comment (foolip) [reply css/properties/mask.json:40]: Actually, I've asked in https://bugs.chromium.org/p/chromium/issues/detail?id=432153 if it's the right bug, I'll link it if the answer is yes.

**Review by foolip — APPROVED** [review]

**Pull request merged**